common/static_modifiers/local_autonomy_multiplicative.txt

```diff
@@ -7,5 +7,5 @@ local_autonomy_multiplicative = {
 	province_trade_power_modifier = -0.5
 	land_forcelimit_modifier = -1.0 # Misleading, only used to avoid errors when applying effect of Estates in code. 
 	naval_forcelimit_modifier = -1.0 # Misleading, only used to avoid errors when applying effect of Estates in code.
-	local_autonomy = -0.10
+	#local_autonomy = -0.10
```

Comment:
I definitely agree with removing this. If anything, there should be a local autonomy penalty at higher autonomy levels, but that would probably make autonomy be too punishing.